lstchain/tools/lstchain_create_calibration_file.py

```diff
@@ -153,7 +153,7 @@ def start(self):
                 if self.simulation:
                     event.r1.tel[tel_id].waveform = (
                             event.r0.tel[tel_id].waveform.astype(float)
-                            - event.mon.tel[tel_id].calibration.pedestal_per_sample[..., np.newaxis]
+                            -np.median(event.mon.tel[tel_id].calibration.pedestal_per_sample, axis=1)[...,np.newaxis, np.newaxis]
```

Comment:
A comment would be good, something like:

```
# estimate offset of each channel from pedestal per pixel values
offset = np.mean(event.mon.tel[tel_id].calibration.pedestal_per_sample, axis=1).round()
event.r1.tel[tel_id].waveform = event.r0.tel[tel_id].waveform.astype(np.float32) - offset[:, np.newaxis, np.newaxis]
```

Not that the waveform should be `np.float32` not `float`, which is the same as `np.float64`